charts/sonarqube-dce/README.md

```diff
@@ -211,8 +211,10 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `searchNodes.startupProbe.timeoutSeconds`                 | StartupProbe timeout delay                                                            | `1`                                                                    |
 | `searchNodes.resources.requests.memory`                   | memory request for Search Nodes                                                       | `2Gi`                                                                  |
 | `searchNodes.resources.requests.cpu`                      | cpu request for Search Nodes                                                          | `400m`                                                                 |
+| `searchNodes.resources.requests.ephemeral-storage`        | storage request for Search Nodes                                                        | `1536M`                                                                |
 | `searchNodes.resources.limits.memory`                     | memory limit for Search Nodes. should not be under 4G                                 | `4096M`                                                                |
 | `searchNodes.resources.limits.cpu`                        | cpu limit for Search Nodes                                                            | `800m`                                                                 |
+| `searchNodes.resources.limits.ephemeral-storage`          | storage limit for Search Nodes                                                        | `512000M`                                                                |
```

Comment:
512G for the ephemeral limit ? is there a specific reason for that ?

Reply:
We will have configmap, secret, volumes counting towards this.

For us the biggest would be plugins ? that are stored under some volumes and can weight few gigs of JAR.

Dont have a strong opinion, i am fine with keeping that way.

Reply:
Ok, I'll keep like this, without setting any strict limit..